patterns/newtype.md

```diff
@@ -1,5 +1,9 @@
 # Newtype
 
+Rust has strong static types. This can be very different than what you are used to if you are coming from a loosely-typed language. Don't worry, though. Once you get used to them, you'll find the types actually make your life easier. Why? Because you are making implicit assumptions explicit.
```

Comment:
How is this paragraph related to "Newtype" pattern?

Reply:
I thought it might be a nice "intro" to the pattern 😅
If you don't like it we can discuss about it.
Anyway as simonsan said maybe it is better to open a new PR.